modules/machine-user-infra-machines-iso.adoc

```diff
@@ -12,13 +12,12 @@ ISO image to create the machines.
 
 * Obtain the URL of the Ignition config file for the compute machines for your
 cluster. You uploaded this file to your HTTP server during installation.
-* Obtain the URL of the BIOS or UEFI {op-system} image file that you uploaded
-to your HTTP server during cluster installation.
 
 .Procedure
 
-. Use the ISO file to install {op-system} on more compute machines. Use the same
-method that you used when you created machines before you installed the cluster:
+. Use the ISO file to install {op-system} on more compute machines. This ISO
+file must be the same one you previously used to install the cluster. Use the
```

Comment:
We can drop the requirement on using the same ISO file; there is a KCS article forthcoming about how to use any version - https://access.redhat.com/solutions/5514051

Reply:
Ack, dropped!